### PR TITLE
Release 6.8

### PR DESCRIPTION
**Pull request by vyakimchik**

### CODE REVIEW

File: authenticity/e-check-diagnose.yml [hunk 1]
```diff
@@ -226,4 +227,5 @@ components:
         - "HOLO_PHOTO_DOCUMENT_OUTSIDE_FRAME"
         - "LIVENESS_DEPTH_CHECK_FAILED"
         - "LAS_INK_INVALID_LINES_FREQUENCY"
-        - "LAST_DIAGNOSE_VALUE"
```
Comment (ikliashchou) [inline authenticity/e-check-diagnose.yml:229]: wrong

File: e-rfid-error-codes.yml [hunk 2]
```diff
@@ -0,0 +1,238 @@
+openapi: 3.0.3
+components:
+  schemas:
+    RFIDErrorCodes:
+      description: Enumeration contains a set of error codes returned by SDK
```
Comment (ikliashchou) [inline e-rfid-error-codes.yml:5]: wrong

File: e-rfid-error-codes.yml [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+      - "RFID_ERROR_LAYER34_COLLISION_TOO_MANY"
+      - "RFID_ERROR_LAYER34_PROTOCOL_B"
+      - "RFID_ERROR_LAYER34_DATA_CONTENTS"
+      - "RFID_ERROR_LAYER34_PROTOCOL"
+      - "RFID_ERROR_LAYER34_GLOBAL_TIME_OUT"
+      - "RFID_ERROR_LAYER34_MIFARE_AUTH"
+      - "RFID_ERROR_LAYER34_SAM_ERROR"
+      - "RFID_ERROR_LAYER34_SAM_COLLISION"
+      - "RFID_ERROR_LAYER34_SAM_ACKNOWLEDGE"
+      - "RFID_Error_Session_BitnessMismatch"
```
Comment (ikliashchou) [inline e-rfid-error-codes.yml:238]: wrong
list is not complete

File: e-security-feature-type.yml [hunk 4]
```diff
@@ -91,6 +92,7 @@ components:
         - "Liveness electronic device"
         - "Liveness OVI"
         - "Barcode size check"
+        - "LASINK Security Feature Check"
```
Comment (ikliashchou) [inline e-security-feature-type.yml:95]: wrong

File: e-security-feature-type.yml [hunk 5]
```diff
@@ -133,4 +135,5 @@ components:
         - "LIVENESS_SCREEN_CAPTURE"
         - "LIVENESS_ELECTRONIC_DEVICE"
         - "LIVENESS_OVI"
-        - "BARCODE_SIZE_CHECK"
\ No newline at end of file
+        - "BARCODE_SIZE_CHECK"
+        - "LAS_INK"
```
Comment (ikliashchou) [inline e-security-feature-type.yml:139]: wrong

File: p-process.yml [hunk 6]
```diff
@@ -373,6 +373,15 @@ components:
           items:
             type: integer
           # default []
+        rfid:
+          type: object
+          description: "Params for the RFID chip processing"
```
Comment (ikliashchou) [inline p-process.yml:378]: wrong

File: p-process.yml [hunk 7]
```diff
@@ -373,6 +373,15 @@ components:
...
+          properties:
+            paSensitiveCodesDisable:
+              description: "A list of passive authentication (PA) sensitive codes that are considered non-critical."
```
Comment (ikliashchou) [inline p-process.yml:381]: wrong

File: rt-status.yml [hunk 8]
```diff
@@ -60,7 +60,7 @@ components:
         - text
       properties:
         overallStatus:
-          description: "Overall optical checks combined status"
+          description: "Summary of all optical results"
```
Comment (ikliashchou) [inline rt-status.yml:63]: wrong, revert

File: e-pa-sensitives-codes.yml [hunk 9]
```diff
@@ -0,0 +1,444 @@
+openapi: 3.0.3
```
Comment (ikliashchou) [inline e-pa-sensitives-codes.yml:1]: wrong file name

File: e-pa-sensitives-codes.yml [hunk 10]
```diff
@@ -0,0 +1,444 @@
...
+components:
+  schemas:
+    ParsingNotificationCodes:
+      description: "A set of codes of non-critical remarks detected during analysis of data structure used during SDK work."
```
Comment (ikliashchou) [inline e-pa-sensitives-codes.yml:5]: wrong

File: e-pa-sensitives-codes.yml [hunk 11]
```diff
@@ -0,0 +1,444 @@
...
+      - "NTF_LDS_CV_CERTIFICATE_PROFILE_INCORRECT_VERSION"
+      - "NTF_LDS_CV_CERTIFICATE_VALIDITY"
+      - "NTF_LDS_CV_CERTIFICATE_NON_CV_CA_DOMAIN_PARAMETERS"
+      - "NTF_LDS_CV_CERTIFICATE_PRIVATE_KEY_INCORRECT_VERSION"
+      - "NTF_LDS_TA_PACE_STATIC_BINDING_USED"
+      - "NTF_LDS_AUTH_ML_SIGNER_INFO_CERTIFICATE_VALIDITY"
+      - "NTF_LDS_AUTH_ML_SIGNER_INFO_CERTIFICATE_ROOT_IS_NOT_TRUSTED"
+      - "NTF_LDS_AUTH_ML_SIGNER_INFO_CERTIFICATE_CANT_FIND_CSCA"
+      - "NTF_LDS_AUTH_ML_SIGNER_INFO_CERTIFICATE_REVOKED"
+      - "NTF_LDS_AUTH_ML_SIGNER_INFO_CERTIFICATE_SIGNATURE_INVALID"
```
Comment (ikliashchou) [inline e-pa-sensitives-codes.yml:444]: list contents wrong

**Review by ikliashchou — CHANGES_REQUESTED** [review]

File: p-process.yml (hunk 7 above)
Comment (vyakimchik) [reply p-process.yml:381]:
```suggestion
              description: "A list of notification codes that should be ignored during passive authentication (PA)"
```

File: e-pa-sensitives-codes.yml (hunk 10 above)
Comment (vyakimchik) [reply e-pa-sensitives-codes.yml:5]:
```suggestion
      description: "The enumeration contains possible values of notification codes returned during the RFID chip processing"
```

File: e-security-feature-type.yml (hunk 4 above)
Comment (vyakimchik) [reply e-security-feature-type.yml:95]:
```suggestion
        - "LASINK"
```

File: e-rfid-error-codes.yml (hunk 2 above)
Comment (vyakimchik) [reply e-rfid-error-codes.yml:5]:
```suggestion
      description: The enumeration contains a set of possible errors that can return during the RFID chip processing
```

File: e-security-feature-type.yml (hunk 5 above)
Comment (vyakimchik) [reply e-security-feature-type.yml:139]:
```suggestion
        - "LASINK"
```

File: p-process.yml (hunk 6 above)
Comment (vyakimchik) [reply p-process.yml:378]:
```suggestion
          description: "Params for the RFID chip reprocessing"
```

File: authenticity/e-check-diagnose.yml (hunk 1 above)
Comment (vyakimchik) [reply authenticity/e-check-diagnose.yml:229]: Fixed in https://github.com/regulaforensics/DocumentReader-web-openapi/pull/108

File: e-pa-sensitives-codes.yml (hunk 11 above)
Comment (vyakimchik) [reply e-pa-sensitives-codes.yml]: Fixed https://github.com/regulaforensics/DocumentReader-web-openapi/pull/108/commits/5046a89f0b8ec85479c630f5353bad7336ba03ea

File: e-parsing-notification-codes.yml [hunk 12]
```diff
@@ -0,0 +1,444 @@
+openapi: 3.0.3
+components:
+  schemas:
+    ParsingNotificationCodes:
+      description: "A set of codes of non-critical remarks detected during analysis of data structure used during SDK work."
```
Comment (vyakimchik) [inline e-parsing-notification-codes.yml:5]:
```suggestion
      description: "The enumeration contains possible values of notification codes returned during the RFID chip processing."
```

File: authenticity/e-check-diagnose.yml [hunk 13]
```diff
@@ -1,229 +1,230 @@
...
+      - WRONG_MRZ_WIDTH
+      - WRONG_MRZ_HEIGHT
+      - WRONG_LINE_POSITION
+      - WRONG_FONT_TYPE
+      - TEXT_POSITION
+      - INVALID_FONT
+      - INVALID_BACKGROUND
+      - LASINK_INVALID_LINES_FREQUENCY
+      - LAST_DIAGNOSE_VALUE
+openapi: 3.0.3
```
Comment (ikliashchou) [inline authenticity/e-check-diagnose.yml]: should be on top

File: e-security-feature-type.yml [hunk 14]
```diff
@@ -1,136 +1,138 @@
...
-        - "Micro text"
-        - "Fluorescent Object"
-        - "Facial landmarks check"
-        - "Facial image presence"
-        - "Facial image absence"
-        - "Liveness screen capture"
-        - "Liveness electronic device"
-        - "Liveness OVI"
-        - "Barcode size check"
+      - Blank element
```
Comment (ikliashchou) [inline e-security-feature-type.yml:52]: why here description without ""?

File: e-text-field-type.yml [hunk 15]
```diff
@@ -1212,6 +1224,19 @@ components:
         - 'DL class code PW valid from'
         - 'DL class code PW valid notes'
         - 'DL class code PW valid to'
+        - 'DL class code EB from'
```
Comment (ikliashchou) [inline e-text-field-type.yml:1227]: why '' instead of ""?

File: e-text-field-type.yml [hunk 16]
```diff
@@ -1212,6 +1224,19 @@ components:
         - 'DL class code PW valid from'
```
Comment (ikliashchou) [inline e-text-field-type.yml:1224]: check all DL codes in enum, description should be the same style everywhere as here

**Review by ikliashchou — CHANGES_REQUESTED** [review]

File: index.yml [hunk 17]
```diff
@@ -3,8 +3,8 @@ info:
   version: 6.7.0
   title: Regula Document Reader Web API
   x-logo:
-    url: 'https://static-content.regulaforensics.com/Icons/Logos/Regula-logo.svg'
-    href: 'https://mobile.regulaforensics.com/'
+    url: "https://static-content.regulaforensics.com/Icons/Logos/Regula-logo.svg"
+    href: "https://mobile.regulaforensics.com/"
```
Comment (vyakimchik) [inline index.yml:7]:
```suggestion
    href: "https://regulaforensics.com"
```

File: authenticity/e-check-diagnose.yml [hunk 18]
```diff
@@ -225,5 +216,17 @@ components:
         - "HOLO_PHOTO_FINISHED_BY_TIMEOUT"
         - "HOLO_PHOTO_DOCUMENT_OUTSIDE_FRAME"
         - "LIVENESS_DEPTH_CHECK_FAILED"
-        - "LAS_INK_INVALID_LINES_FREQUENCY"
-        - "LAST_DIAGNOSE_VALUE"
\ No newline at end of file
+        - "SCREEN_CAPTURE_DETECTED"
+        - "ELECTRONIC_DEVICE_DETECTED"
+        - "WRONG_SYMBOL_BACKGROUND"
+        - "WRONG_BACKGROUND"
```
Comment (ikliashchou) [inline authenticity/e-check-diagnose.yml:222]: wrong

File: e-security-feature-type.yml [hunk 19]
```diff
@@ -133,4 +136,6 @@ components:
         - "LIVENESS_SCREEN_CAPTURE"
         - "LIVENESS_ELECTRONIC_DEVICE"
         - "LIVENESS_OVI"
-        - "BARCODE_SIZE_CHECK"
\ No newline at end of file
+        - "BARCODE_SIZE_CHECK"
+        - "LASINK"
+        - "ELECTRONIC_DEVICE"
```
Comment (ikliashchou) [inline e-security-feature-type.yml:141]: wrong

File: e-lcid.yml [hunk 20]
```diff
@@ -1,436 +1,432 @@
 openapi: 3.0.3
```
Comment (ikliashchou) [inline e-lcid.yml:1]: I propose not to touch this one.
Revert enum changes

File: e-text-field-type.yml [hunk 21]
```diff
@@ -608,1211 +607,1249 @@ components:
...
+        - "DL_CLASS_CODE_EB_TO"
+        - "DL_CLASS_CODE_EC_FROM"
+        - "DL_CLASS_CODE_EC_NOTES"
+        - "DL_CLASS_CODE_EC_TO"
+        - "DL_CLASS_CODE_EC1_FROM"
+        - "DL_CLASS_CODE_EC1_NOTES"
+        - "DL_CLASS_CODE_EC1_TO"
+        - "PLACE_OF_BIRTH_CITY"
+        - "YEAR_OF_EXPIRY"
+        - "GRANDFATHER_NAME_MATERIAL"
```
Comment (ikliashchou) [inline e-text-field-type.yml:1855]:
```suggestion
        - "GRANDFATHER_NAME_MATERNAL"
```

**Review by ikliashchou — CHANGES_REQUESTED** [review]

**Review by ikliashchou — APPROVED** [review]

**Pull request merged**